matrixbot/plugins/wkbotsfeeder.py

```diff
@@ -115,47 +117,83 @@ def dispatch(self, handler=None):
             except Exception as e:
                 self.logger.error("WKBotsFeederPlugin got error in builder %s: %s" % (builder_name,e))
 
-
-    def command(self, sender, room_id, body, handler):
-        self.logger.debug("WKBotsFeederPlugin command")
-        return
+    def command(self, sender, room_id, body, handler = None):
+        self.logger.debug("WKBotsFeederPlugin command: %s" % body)
+        if len(body) == 0:
+            return
+        command = body.split()
+        if command[0] != self.settings["name"]:
+            return
+        if len(command) > 1 and command[1].lower() == "mute":
+            self.command_mute(sender, room_id, command[2:], handler)
+
+    def command_mute(self, sender, room_id, command, handler):
+        # Command: mute <builder-name> [ON|OFF].
+        if len(command) == 0:
+            self.help(sender, room_id, handler)
+            return
+        builders = self.settings["builders"]
+        builderName = command[0]
+        if builderName not in builders.keys():
+            self.help(sender, room_id, handler)
+            return
+        value = (command[1] if command[1:] else "on").lower()
+        if value not in ["on", "off"]:
+            self.help(sender, room_id, handler)
+            return
+        if value == "on":
+            builders[builderName]["mute"] = True
+        else:
+            builders[builderName].pop("mute")
+        self.bot.send_html(room_id, "<p>Room %s was set to mute %s</p>" % (room_id, value))
 
     def help(self, sender, room_id, handler):
+        if not handler:
+            return
         self.logger.debug("WKBotsFeederPlugin help")
-        return
+        message = "<p>mute <builder-name> [ON|OFF]<p>"
```

Comment:
Did you tried this? I  think <p> are not needed. Other plugins don't use that.

Reply:
... for example the plugin/trac.py uses: `message = "%(name)s create This is the issue summary\n" % self.settings`